src/encoded/types/biosample.py

```diff
@@ -119,6 +119,66 @@ def treatments_summary(self, request, treatments=None):
             return ' and '.join(sorted([t for t in treat_list if t]))
         return 'None'
 
+    @calculated_property(schema={
+        "title": "Tissue, Organ/System Info",
+        "description": "Useful faceting info for biosample",
+        "type": "object",
+        "properties": {
+            "tissue_source": {
+                "type": "string"
+            },
+            "organ_system": {
+                "type": "array",
+                "items": {
+                    "type": "string"
+                }
```

Comment:
Codacy found an issue: [Method could be a function](https://app.codacy.com/manual/4dn/fourfront/pullRequest?prid=6380012)